app/archivesspace/archivesspace.controller.js

```diff
@@ -72,10 +92,29 @@
             level: function() {
               return '';
             },
+            start_date: function() {
+              return new Date;
+            },
+            end_date: function() {
+              return new Date;
+            },
+            date_expression: function() {
+              return '';
+            },
+            note: function() {
+              return '';
+            },
           },
         });
         form.result.then(function(result) {
           result.levelOfDescription = result.level;
+          result.notes = [{
+            type: 'odd',
+            content: result.note,
+          }];
+          delete result['note'];
```

Comment:
`$scope.edit` and `$scope.add_child` see to duplicate a lot of code. Is there a way to merge them?